README.md

```diff
@@ -301,6 +305,28 @@ These methods can be used from the [API](./actors/api.go) as well as from the [P
 This offers simple scalability, partitioning (sharding), and re-balancing out-of-the-box. Go-Akt nodes are automatically discovered. See [Clustering](#clustering).
 Beware that at the moment, within the cluster the existence of an actor is unique.
 
+When levereging cluster re-balancing, leaving the supervisor to re-instatiate an actor in case of node fail, actors should be created as plain structs without parameters. This is required because when the supervisor re-creates needed actors will not be aware of those parameters.
```

Comment:
I don't think this note should be here. It should be rather under `Redeployment` section. Also there is no concept of supervisor in the clustering feature. Once cluster mode is enabled, actors are redeployed whenever the cluster topology changes and the members quorum respected.